src/DI/ContainerBuilder.php

```diff
@@ -480,6 +520,18 @@ public function generateClasses($className = 'Container', $parentName = NULL)
 			}
 		}
 
+		foreach ($aliases as $alias => $service) {
```

Comment:
Hm... originally it was to exclude aliases for non-defined services, but now I realize that we might actually want to keep those. And omitting the check for sane value of alias is probably no big deal.